Makefile

```diff
@@ -753,9 +752,16 @@ devbox-grpc:
 		lib/multiplexer/test/ping.proto \
 		lib/web/envelope.proto
 
+	# gogoproto is not aware of go modules, so generated go files fail to read
+	# from the correct api import path for cast types and custom types unless 
+	# explicitly told where to look.
+	$(eval API_IMPORT_PATH=$(shell go run build.assets/go-modules/print-api-import-path/main.go))
+
 	protoc -I=.:$$PROTO_INCLUDE \
 		--proto_path=api/client/proto \
-		--gogofast_out=plugins=grpc:api/client/proto \
+		--gogofast_out=plugins=grpc,\
+Mgithub.com/gravitational/teleport/api/types/events/events.proto=$$API_IMPORT_PATH/types/events:\
+$$GOPATH/src \
```

Comment:
This syntax confused me a bit at first. Can you add a comment explaining that `M` flag is used for specifying the Go import path for proto files?